deployment/base/crds/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Typo in commit msg s/cri-resmgr/nri-resmgr/

Reply:
Ach 🤦‍♂️ Well spotted, fixed